docker.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+xhost +
+docker run \
+    --name ucsd_ros \
+    --runtime nvidia \
+    -it \
+    --rm \
+    --privileged \
+    --net=host \
+    -e DISPLAY=$DISPLAY \
+    -v /dev/bus/usb:/dev/bus/usb \
+    --device-cgroup-rule='c 189:* rmw' \
+    --device /dev/video0 \
+    --volume='/dev/input:/dev/input' \
+    --volume='/home/jetson/.Xauthority:/root/.Xauthority:rw' \
+    --volume='/tmp/.X11-unix/:/tmp/.X11-unix' \
+    --volume='/home/jetson/projects/robocar:/home/jetson/projects/robocar' \
+    ghcr.io/ucsd-ecemae-148/donkeycontainer:ros
```

Comment:
should we use `ghcr.io/triton-ai/robocar:foxy-x86` here?

Reply:
no that's for arm arch

Reply:
should find some time to work on the arm arch image